### ISSUE TITLE
Code Signing Certificate for MSI file

### ISSUE THREAD

**Issue by NowinskiK** [SQLPlayer/SCD-Merge-Wizard#12]
Would be nice to have a robust installation file signed with a certificate instead of seeing that window when attempting to install the app from MSI file:
![image](https://user-images.githubusercontent.com/25029325/89581440-e331ac00-d82e-11ea-9da7-f8ad39aa320f.png)

But the certificate is not cheap, so I'm wondering how to gather the required amount of money and buy a certificate for at least 3 years. Any ideas?

Price comparison:
https://www.sslpoint.com/eu/code-signing/
Minimum is $69.00/Year x3 = $207

**Comment by jwk6** [comment]
I checked to see if https://letsencrypt.org/ offers Code Signing certificates.  They offer FREE TLS/SSL certificates. 

But alas they do *not* support code signing.  Here's some discussion on the topic -https://community.letsencrypt.org/t/do-you-support-code-signing/370/10

**Comment by jwk6** [comment]
Some useful information re: why code signing certs are not free - https://comodosslstore.com/resources/free-code-signing-certificate/

**Comment by NowinskiK** [comment]
Yeah, I've seen all those things. There is no free certificate and I'm fine with that. Hence, I came up with an idea like this:  
https://github.com/SQLPlayer/SCD-Merge-Wizard/wiki/Thank-You
**To everyone:** Please read and donate if you wish!

**Comment by jwk6** [comment]
@NowinskiK I had another thought.  What about distributing through the Microsoft Store?

https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/desktop-to-uwp-distribute 

"You don't have to sign your application before you submit it to the Store."

**Comment by NowinskiK** [comment]
Cool. That could be another option. I will consider that when have some free time.
